common/model/bundle.c

```diff
@@ -12,6 +12,60 @@
 
 static UT_icd bundle_transactions_icd = {sizeof(iota_transaction_t), 0, 0, 0};
 
+static retcode_t validate_signature(bundle_transactions_t const *const bundle,
```

Comment:
you're validating multiple signatures at once, so cal this `validate_signatures`?

Reply:
sure